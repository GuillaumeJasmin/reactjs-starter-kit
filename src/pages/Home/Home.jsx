import React from 'react';
import classes from './Home.css';

export default () => (
  <div className={classes.container}>
    <div>Welcome on Home</div>
  </div>
);
