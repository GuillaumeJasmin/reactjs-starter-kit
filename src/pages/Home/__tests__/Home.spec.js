import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Home from '../Home.jsx';

/* global describe, it, expect */
describe('Home', () => {
  it('must be contain good text', () => {
    const home = shallow(<Home />);
    expect(home.text()).toEqual('Welcome on Home');
  });

  it('must always be the same', () => {
    const snapshot = renderer.create(<Home />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
