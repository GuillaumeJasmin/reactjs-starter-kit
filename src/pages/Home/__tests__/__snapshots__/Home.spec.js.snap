// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home must always be the same 1`] = `
<div
  className="container"
>
  <div>
    Welcome on Home
  </div>
</div>
`;
